### DISCUSSION TITLE
Stripping/removing markdown -> raw text

### DISCUSSION THREAD

**Discussion by RemyMachado** [remarkjs/remark#784] [Q&A] [answered]
### Is there a way to remove the markdown from a string?

For instance:
`**text** -> text`
instead of:
`**text** -> <em>text</em>`

If so, it would be great to mention it on the Readme.md.
Thanks in advance for your response.

**Comment by wooorm** [comment, marked as answer]
There are many ways. Could you expand on your question?

**Reply by RemyMachado** [reply]
Yes sorry. I didn't pay attention in the first place, that I had been redirected on the **remark** github.

I would like to know if there is a function (e.g. `rawText()`) packed with **react-markdown** that would get rid of the markdown.

My use case: I'm using the markdown string somewhere, but I need to display this exact same string elsewhere without the markdown.

**Reply by wooorm** [reply]
Can you expand on your use case? What is raw text? What about paragraphs? What about tables? What about lists?

**Reply by RemyMachado** [reply]
I want to remove every symbols that markdown would normarlly interpret.

For instance:

```
## Overview

* **Follows** [CommonMark](https://commonmark.org)
```

would give:

```
Overview
Follows CommonMark
```

or:

```
Overview
Follows
```

or:
```
Overview
Follows CommonMarkhttps://commonmark.org
```

Any of these outputs would be fine for me.

My use case is a title input, where the user can put bold/italic tags. He shouldn't put anything fancy, so I want to retrieve only the text.

**Reply by wooorm** [reply]
`react-markdown` has a different goal: rendering markdown as React nodes.
[`strip-markdown`](https://github.com/remarkjs/strip-markdown) does some of what you want.
[`rehype-sanitize`](https://github.com/rehypejs/rehype-sanitize) lets you choose which elements are (not) allowed.
[`hast-util-to-text`](https://github.com/syntax-tree/hast-util-to-text) can turn a hast element into nice plain test that you probably want. I’d suggest this route.
